app/validators/answers/option_answer_validator.py

```diff
@@ -26,6 +26,9 @@ class OptionAnswerValidator(AnswerValidator):
     DYNAMIC_OPTIONS_REFERENCES_NON_CHECKBOX_ANSWER = (
         "Dynamic options references non Checkbox answer"
     )
+    DYNAMIC_OPTIONS_SOURCE_IDENTIFIER_AND_OPTION_LABEL_FROM_VALUE_MISMATCH = (
+        "Dynamic options source identifier and option label from value do not match"
```

Comment:
I left as is because there is something mis-aligned with format and linting at the minute. That change make the line 163 which is fine so I just bracket it and put it on a new line, format puts it back on the same line, then linting breaks.

Reply:
Okay, odd, fortunately you can break down strings in python on multiple lines. I tried this and black and linting are happy.
```py
    DYNAMIC_OPTIONS_SOURCE_IDENTIFIER_AND_OPTION_LABEL_FROM_VALUE_MISMATCH = (
        "Dynamic options source identifier and "
        "option label from value answer_id do not match"
    )
```